cmd/river/go.mod

```diff
@@ -10,6 +10,8 @@ replace github.com/riverqueue/river/riverdriver/riverdatabasesql => ../../riverd
 
 replace github.com/riverqueue/river/riverdriver/riverpgxv5 => ../../riverdriver/riverpgxv5
 
+replace github.com/riverqueue/river/rivertype => ../../rivertype
```

Comment:
Need to remember to re-comment all these `replaces` in the CLI module before final release.